server/routes/title.js

```diff
@@ -1,37 +1,63 @@
 const router = require('express').Router();
 const Promise = require('bluebird');
 const faker = require('faker');
+const axios = require('axios');
+let seedUri = 'http://localhost:3001/api/scaled-seed/';
 
 
 let Title = require('../../db/title.model');
 const mongoose = require('mongoose');
 const saveEnrolled = require('./enrolled').saveEnrolled;
-const exampleDataGenerator = require('../example.data').exampleDataGenerator;
+const generator = require('../example.data');
 
-mongoose.connect('mongodb://localhost:27017/tittle');
 
 let saveTittle = (cb) => {
-  let tittleData = exampleDataGenerator();
+  let tittleData = generator.exampleDataGenerator();
 
   Title.insertMany(tittleData)
     .then((seededData) => cb(null, seededData))
     .catch((err => cb(`SEEDING TITTLES ERROR = ${err}`, null)));
 };
-
 saveTittle = Promise.promisify(saveTittle);
 
-const seed = async (cb) => {
+
+let writeScaledTittles = (currentCounts, cb) => {
+  let tittleData = generator.generateScaledTittles(currentCounts);
+  console.log('writting tittles to db...');
+
+  Title.insertMany(tittleData)
+    .then((seededData) => cb(null, seededData))
+    .catch((err => cb(`SEEDING TITTLES ERROR = ${err}`, null)));
+};
+writeScaledTittles = Promise.promisify(writeScaledTittles);
+
+
+let scaledSeed = async (currentCounts, cb) => {
+  console.time('seed');
+
+  try {
+    await writeScaledTittles(currentCounts).then((seededTittles) => console.log('seededTittles = ', seededTittles));
+    console.timeEnd('seed');
+    console.log('data successfully seeded');
+    cb(null, 'data seeded');
+
+  } catch(e) { cb(e, null); }
+};
+scaledSeed = Promise.promisify(scaledSeed);
```

Comment:
Just a thought on these two functions - since you control them, you may consider just straight up writing them as a Promise instead of using callbacks and then promisifying them with Bluebird. It would look something like this where you're mainly just replacing the callbacks with either `resolve` or `reject` and wrapping the whole thing in `new Promise()`:
```javascript
let scaledSeed = async (currentCounts, cb) => {
  console.time('seed');

  return new Promise((resolve, reject) => {
    try {
      await writeScaledTittles(currentCounts).then((seededTittles) => console.log('seededTittles = ', seededTittles));
      console.timeEnd('seed');
      console.log('data successfully seeded');
      resolve('data seeded');

    } catch(e) { reject(e); }
  })
};
```
Of course, if this is legacy code and you just wanted to convert it easily, it's still perfectly valid!

Reply:
Yes thank you for the suggestion. Your suggestion seems to be a more straight forward way of promisifying it, which is very useful when I don't or might not have access to bluebird. 
 
I can't quite tell for sure if this is 100% accurate, but this post is basically saying bluebird's promise is faster and consumes less memory than JS's native resolve / reject promise:
https://blog.kuzzle.io/bluebird-native-async_await-javascript-promises-performances-2020